uwblueprint/abtc
# Create route and implement getServiceRequests

## Notion ticket link

[Create route to getServiceRequests](https://www.notion.so/uwblueprintexecs/Create-route-to-getServiceRequests-55f86f551fd44596b50d421682f071ee?pvs=4)



## Implementation description
* Added a route for service request GET requests
* With no requestId parameter, the endpoint returns all serviceRequests (.../serviceRequests/)
<img width="855" alt="image" src="https://github.com/uwblueprint/abtc/assets/67832534/65bddeb9-4061-4f75-8bf9-d072f18ecd7d">

* With a valid requestId parameter, the endpoint queries service requests by ID to return the desired service request (e.g. .../serviceRequests/?requestId=65c5788ef191c0a3acf65ec6)
<img width="854" alt="image" src="https://github.com/uwblueprint/abtc/assets/67832534/ac6f5201-87b7-40d6-beb4-3dc7c7694d2e">




## Steps to test
1. Run docker and build
2. Go to localhost:5000/serviceRequests/ - you should see a JSON list of all service requests!
3. Go to localhost:5000/serviceRequests/?requestId=65c5788ef191c0a3acf65ec6 - you should see the service request with id 65c5788ef191c0a3acf65ec6
4. Similarly, localhost:5000/serviceRequests/?requestId={anyValidRequestId} should return the service request with corresponding ID



## What should reviewers focus on?
* 


## Checklist
